toolchain/sem_ir/formatter.cpp

```diff
@@ -572,6 +582,26 @@ class Formatter {
     }
   }
 
+  auto FormatNameScope(NameScopeId id, llvm::StringRef separator,
+                       llvm::StringRef prefix) -> void {
+    // Name scopes aren't kept in any particular order. Sort the entries before
+    // we print them for stability and consistency.
+    std::vector<std::pair<NodeId, StringId>> entries;
```

Comment:
```suggestion
    llvm::SmallVector<std::pair<NodeId, StringId>> entries;
```

Just suggesting for consistency, the rule of thumb to use SmallVector and not vector in toolchain code.

Reply:
(This is pre-existing, just moved from below.) Done.